org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/watchdog/DeviceWatchdogService.java

```diff
@@ -79,5 +79,9 @@ public IDeviceController create(IPausableDevice<?> device) {
 	public IDeviceWatchdog getWatchdog(String name) {
 		return templates.get(name);
 	}
+	@Override
+	public List<String> getRegisteredNames() {
+		return new ArrayList<String>(templates.keySet());

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)